problems/objects.js

```diff
@@ -61,7 +78,11 @@ const appleSet = () => {};
  *
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = (appleCountByName) => {
+  // Object.values(appleCountByName)
```

Comment:
Remember to delete dead code. 
Also we don't need that sum variable below, we can just return directly b/c we're not using sum for anything.